src/deps_info.json

```diff
@@ -57,7 +57,6 @@
   "bind": ["htonl", "htons", "ntohs"],
   "connect": ["htonl", "htons", "ntohs"],
   "socket": ["htonl", "htons", "ntohs"],
-  "sleep": ["usleep"],
```

Comment:
what is this for?

Reply:
AFAICT, both sleep and usleep are defined in JS.  Neither of them is built in C/C++ code. 

One of the tests was failing because usleep was being added to EXPORTED_FUNCTIONS due to this deps_info.  Does that make sense.    I think this line says that the sleep JS function depedns on the usleep native function right?  That doesn't seem to true.  Please correct me if I'm wrong here.

Reply:
I see `usleep` defined in JS in `src/library_pthread_stub.js`, but that's when not using pthreads. With pthreads, the impl is in `system/lib/pthread/library_pthread.c`. So this is a weird case where the impl could be in JS or native. One option might be to move the JS one into native code.